### PR TITLE
Feature/credits modal

### PR DESCRIPTION
**Pull request by dhruv-divtech**

### CODE REVIEW

File: src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  top: '22px',
+  padding: '0',
+  right: '20px'
+}));
+
+export const BalanceInfoBox = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  gap: theme.spacing(1),
+  alignItems: 'center',
+  paddingRight: '35px'
```
Comment (WorkspaceChetan) [inline src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx:44]: use theme.spacing

File: src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  flexDirection: 'column'
+}));
+
+export const HeadingContainer = styled(Box)(() => ({
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+  height: '100%',
+  width: '100%',
+  padding: '20px 24px',
```
Comment (WorkspaceChetan) [inline src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx:22]: use theme.spacing

File: src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+export const ImagMainContainer = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  width: '100%',
+  height: '100%',
+  gap: theme.spacing(3),
+  flexWrap: 'wrap',
+  overflow: 'hidden',
+  marginTop: '48px',
```
Comment (WorkspaceChetan) [inline src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx:69]: use theme.spacing

File: src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+  [theme.breakpoints.up('md')]: {
+    justifyContent: 'space-between'
+  },
+  [theme.breakpoints.down('sm')]: {
+    marginTop: '32px'
```
Comment (WorkspaceChetan) [inline src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx:74]: use theme.spacing

File: src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+  [theme.breakpoints.up('md')]: {
+    flexBasis: 'calc(33.333% - 16px)'
+  }
+}));
+
+export const MainImagContainer = styled('img')(({ theme }) => ({
+  display: 'flex',
+  width: '100%',
+  height: '100%',
+  borderRadius: '8px',
```
Comment (WorkspaceChetan) [inline src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx:93]: use theme.spacing

File: src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+  left: '75%',
+  transform: 'translate(-50%, -50%)',
+  display: 'flex',
+  width: '100%',
+  alignItems: 'center',
+  paddingTop: theme.spacing(0.5),
+  paddingBottom: theme.spacing(0.5),
+  paddingRight: theme.spacing(1),
+  paddingLeft: theme.spacing(1),
+  borderRadius: '4px'
```
Comment (WorkspaceChetan) [inline src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx:110]: use theme.spacing

File: src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+  paddingLeft: theme.spacing(1),
+  borderRadius: theme.spacing(0.5),
+  gap: theme.spacing(0.5)
+}));
+
+export const CreditCardText = styled(UINewTypography)(({ theme }) => ({
+  fontSize: '20px !important',
+  fontWeight: '500 !important',
+  lineHeight: '24px !important',
+  marginLeft: '6px'
```
Comment (WorkspaceChetan) [inline src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx:131]: use theme.spacing

File: src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx (hunk 1 above)
Comment (RitikMehta24) [reply src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx:44]: resolved

File: src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx (hunk 2 above)
Comment (RitikMehta24) [reply src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx:22]: resolved

File: src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx (hunk 3 above)
Comment (RitikMehta24) [reply src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx:69]: RESOLVED

File: src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx (hunk 4 above)
Comment (RitikMehta24) [reply src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx:74]: resolved

File: src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx (hunk 5 above)
Comment (RitikMehta24) [reply src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx:93]: resolved

File: src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx (hunk 6 above)
Comment (RitikMehta24) [reply src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx:110]: resolved

File: src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx (hunk 7 above)
Comment (dhruv-divtech) [reply src/views/protectedViews/Credites/ModelCredits/Credits.styled.tsx]: resolved

**Pull request merged**